com.reprezen.swagedit/plugin.xml

```diff
@@ -166,4 +167,41 @@
       </hyperlinkDetector>
    </extension>
 
+   <extension
+         point="org.eclipse.ui.commands">
+      <command
+            description="Open Quick Outline"
+            id="com.reprezen.swagedit.keybinding.quickoutline"
```

Comment:
Thank you for adding key binding for the command, except I am not sure why we need the word "keybinding" in commandId: `com.reprezen.swagedit.keybinding.quickoutline`